sdk/resourcehealth/Azure.ResourceManager.ResourceHealth/api/Azure.ResourceManager.ResourceHealth.netstandard2.0.cs

```diff
@@ -247,35 +245,32 @@ public partial class TenantResourceHealthEventImpactedResourceCollection : Azure
-        public static Azure.ResourceManager.ResourceHealth.Models.MetadataSupportedValueDetail MetadataSupportedValueDetail(string id = null, string displayName = null, System.Collections.Generic.IEnumerable<string> resourceTypes = null) { throw null; }
-        public static Azure.ResourceManager.ResourceHealth.Models.RecommendedAction RecommendedAction(string action = null, System.Uri actionUri = null, string actionUrlText = null) { throw null; }
-        public static Azure.ResourceManager.ResourceHealth.Models.ServiceImpactingEvent ServiceImpactingEvent(System.DateTimeOffset? eventStartOn = default(System.DateTimeOffset?), System.DateTimeOffset? eventStatusLastModifiedOn = default(System.DateTimeOffset?), string correlationId = null, string statusValue = null, Azure.ResourceManager.ResourceHealth.Models.ServiceImpactingEventIncidentProperties incidentProperties = null) { throw null; }
-        public static Azure.ResourceManager.ResourceHealth.Models.ServiceImpactingEventIncidentProperties ServiceImpactingEventIncidentProperties(string title = null, string service = null, string region = null, string incidentType = null) { throw null; }
-        public static Azure.ResourceManager.ResourceHealth.Models.StatusActiveEvent StatusActiveEvent(string title = null, string description = null, string trackingId = null, System.DateTimeOffset? startOn = default(System.DateTimeOffset?), string cloud = null, Azure.ResourceManager.ResourceHealth.Models.SeverityValue? severity = default(Azure.ResourceManager.ResourceHealth.Models.SeverityValue?), Azure.ResourceManager.ResourceHealth.Models.StageValue? stage = default(Azure.ResourceManager.ResourceHealth.Models.StageValue?), bool? published = default(bool?), System.DateTimeOffset? lastModifiedOn = default(System.DateTimeOffset?), System.Collections.Generic.IEnumerable<Azure.ResourceManager.ResourceHealth.Models.EmergingIssueImpact> impacts = null) { throw null; }
-        public static Azure.ResourceManager.ResourceHealth.Models.StatusBanner StatusBanner(string title = null, string message = null, string cloud = null, System.DateTimeOffset? lastModifiedOn = default(System.DateTimeOffset?)) { throw null; }
-        public static Azure.ResourceManager.ResourceHealth.Models.Update Update(string summary = null, System.DateTimeOffset? updateOn = default(System.DateTimeOffset?)) { throw null; }
+    public partial class AvailabilityStateRecentlyResolved
```

Comment:
Please consider prepend the RP name, this name is a bit generic

Reply:
fixed